### PR TITLE
Basic information about therapist example photo

### PR DESCRIPTION
**Pull request by GoralTomaszGorski**
If it is possible please check aboutMe.html. 
I'm sure you are able to improve it

### CODE REVIEW

**Comment by gorskimarcin96** [comment]
i added few styles. i made gif with presentation, but quality is very bad:
ezgif-4-02f4dc7cf0

i am thinking what i can make with this section, but i dont have idea for this moment. maybe i will add something tomorrow
image

current full page screen (i prupose open screen in new tab):
Psychoterapia-Ewa-Górska

File: src/main/resources/templates/aboutMe.html [hunk 1]
```diff
@@ -6,49 +6,67 @@
...
+                    Opiekuńczo-Leczniczym jak i <b>Powiatowym Domu Samopomocy</b> udzielałam wsparcia indywidualnego i
+                    prowadziłam diagnostykę pacjentów.<br>W ostatnim czasie pracowałam w <b>Szpitalu (SPZOZ) w Kraśniku</b>
+                    na Oddziale Neurologii z Pododdziałem Udarowym, Oddziale Rehabilitacji oraz Oddziale Dziennym
+                    Psychiatrycznym prowadząc terapię indywidualną i grupową.
+                </p>
             </div>
+            <div class="col-25p">
+                <div class="flip-box rounded cursor-pointer float-right">
+                    <div class="flip-box-inner rounded">
+                        <div class="flip-box-front">
```
Comment (gorskimarcin96) [inline src/main/resources/templates/aboutMe.html:46]: i write this tekst for your information. good name for images is important for position page for google. Attribute "alt" is important too. now i think, i will make rename file to "Ewa Górska Kraśnik Terapeuta.png" tomorrow

Comment (gorskimarcin96) [reply src/main/resources/templates/aboutMe.html:46]: i commented this line: `Ewa Górska - Kraśnik Terapeuta`

**Comment by gorskimarcin96** [comment]
uncomfortable is styling page in java, because we must wait few time for building app

File: src/main/resources/templates/aboutMe.html (hunk 1 above)
Comment (GoralTomaszGorski) [reply src/main/resources/templates/aboutMe.html:46]: probably that because of that google robot can fiend easier during the searching

**Comment by GoralTomaszGorski** [comment]
such a great job thank you Marcin!!:):)

**Pull request merged**